pkgs/os-specific/linux/cpupower/gui.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, stdenv, fetchFromGitHub
+, desktop-file-utils
+, gettext
+, glib
+, gobject-introspection
+, gtk3
+, hicolor-icon-theme
+, meson
+, ninja
+, pkg-config
+, python3
+, systemd
+, wrapGAppsHook
+}:
+
+let
+  # use pythonEnv, so that patchShebang fixup will choose this interpreter
+  pythonEnv = python3.withPackages(ps: with ps; [
+    pygobject3
+    dbus-python
+  ]);
+in
+
+stdenv.mkDerivation rec {
+  pname = "cpupower-gui";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "vagnum08";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1rrzlcmspill7qivv3bg8v810ppj4wxa9jcf2ddrjqcbq4lfxayk";
+  };
+
+  nativeBuildInputs = [
+    hicolor-icon-theme # needed for postinstall script
+    desktop-file-utils # needed for update-desktop-database
+    meson
+    ninja
+    glib # needed for glib-compile-scemas
```

Comment:
this should be in `buildInputs`, it's a runtime dependency. but since it's used for `glib-compile-schemas` during the build it is also native so it needs to be in both. also `scemas` is a typo